sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -240,5 +279,9 @@ FULL OUTER JOIN
   metrics_ping AS metrics
 USING
   (client_id)
+LEFT JOIN
+  activations
+USING
+  (client_id)
```

Comment:
are clients guaranteed to only be present once in `activations`?

Reply:
Indeed, good spotting. I counted 168 client_ids with more than one activation record on different dates, I'm using the most recent one. Also considering the case where a client_id happens to have more than one activation on the same day.